tests/foreman/ui/test_host.py

```diff
@@ -17,46 +18,72 @@ def test_create_host(self):
         @Assert: Host is created
 
         """
+        host = entities.Host()
+        host.create_missing()
+        os_name = u'{} {}'.format(
```

Comment:
Bah, nevermind, i got it wrong. Looks like the comment was about unnecessary ```str()```, not about prefered ```u'{}'``` format.